bokeh/_templates/autoload_js.js

```diff
@@ -26,31 +26,38 @@ calls it with the rendered model.
   if (typeof (window._bokeh_onload_callbacks) === "undefined"){
     window._bokeh_onload_callbacks = [];
   }
-  function load_lib(url, callback){
+  function load_libs(js_urls, callback) {
     window._bokeh_onload_callbacks.push(callback);
-    if (window._bokeh_is_loading){
+    if (window._bokeh_is_loading > 0) {
       console.log("Bokeh: BokehJS is being loaded, scheduling callback at", new Date());
       return null;
     }
     console.log("Bokeh: BokehJS not loaded, scheduling load and callback at", new Date());
-    window._bokeh_is_loading = true;
-    var s = document.createElement('script');
-    s.src = url;
-    s.async = true;
-    s.onreadystatechange = s.onload = function(){
-      {%- for file in css_files %}
-      Bokeh.embed.inject_css("{{ file }}");
-      {%- endfor %}
-      window._bokeh_onload_callbacks.forEach(function(callback){callback()});
-    };
-    s.onerror = function(){
-      console.warn("failed to load library " + url);
-    };
-    document.getElementsByTagName("head")[0].appendChild(s);
+    window._bokeh_is_loading = js_urls.length;
+    for (i = 0; i < js_urls.length; i++) {
+      url = js_urls[i];
+      var s = document.createElement('script');
+      s.src = url;
+      s.async = true;
+      s.onreadystatechange = s.onload = function() {
```

Comment:
we assign to both onload and onreadystatechange, which looks like it will call our callback twice on IE9; this page suggests a different pattern to handle all browsers: https://msdn.microsoft.com/en-us/library/hh180173(v=vs.85).aspx
Or if we don't care about IE9 I guess we can simply not use onreadystatechange...